test_scripts/Smoke/commonSmoke.lua

```diff
@@ -322,6 +319,27 @@ function commonSmoke.putFile(params, pAppId, self)
   mobileSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
 end
 
+function commonSmoke.AppType(HMIType)
+  for _,v in pairs(HMIType) do
+    if
+      v == "NAVIGATION" then
+      commonSmoke.HMITypeStatus["NAVIGATION"] = true
+    end
+  end
+end
+
+function commonSmoke.GetAudibleState(pAppId)
+  if not pAppId then pAppId = 1 end
+  if config["application" .. tostring(pAppId)].registerAppInterfaceParams.isMediaApplication == true or
```

Comment:
@GetmanetsIrina 'tostring()' function can be omitted here and in line 338

Reply:
@dboltovskyi, updated.